static/src/js/components/Tree/Tree.js

```diff
@@ -49,7 +49,7 @@ export const Tree = ({
       variables,
       onUpdateFinished: forceUpdate
     })
-  }, []) // Only execute this useEffect once on initial render
+  }, [variables]) // Update TreeModel when the list of variables changes
```

Comment:
This does appear to fix the problem in the tree view, but I don't think it is the best way to fix this problem. Variables in a Tree will never change, so the fact this works means the Tree component is hanging around between the change of the access method, which would lead to unintended side effects like this bug. 

You can also this this same behavior in your test SIT collection by selecting an output format, then switching access methods and the output format will not have changed (or reset). I'm not sure if this is actually a problem because the options between those access methods are the same.

You can force React to re-render a component by adding the `key` prop. When the value you give it changes it will re-render the component. There is probably a more correct way to accomplish the same thing, but just wanted to note those thoughts about this particular solution

Reply:
I agree thank you for the `key` I'm actively looking for a "better" refactor but, I have updated branch to pass the key with a unique identifier to include both the `id` of the service and the `view` so that it re-renders on a change to either